Automattic/pocket-casts-ios
# Refactor Filters List view from storyboard to swiftui view

Project: https://github.com/Automattic/pocket-casts-ios/issues/23

Refactor the top filters view from storyboard controller to use SwiftUI.

![Simulator Screenshot - Apple Watch Series 9 (45mm) - 2024-03-16 at 18 03 29](https://github.com/Automattic/pocket-casts-ios/assets/651601/c6d36ae4-9911-4a39-b3a6-64d3e55ac524)


## To test

- Start the WatchOS app
 -  Select the scheme Pocket Casts Watch App
 - Start on a device directly ( XCode 15.2 has a bug that the pairing options iPhone + watch dont' work correctly)
- Select one of the sources ( Watch or Phone)
- Tap on filters
- Check that the filters show correctly, and counters are correct
  - 0 episodes, shouldn't show a count
  - 1 to 99 episodes, should show the number
  - more than 99 should show 99+
- Tap on one filter and see if you are directed to the correct list of episodes on the filter

 

## Checklist
